docker-compose.yaml

```diff
@@ -8,3 +8,12 @@ services:
     environment:
       APPINSIGHTS_CLOUDROLE: fcp-fd-comms-local
       APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
+      MESSAGE_HOST: ${MESSAGE_HOST}
+      MESSAGE_USER: ${MESSAGE_USER}
+      MESSAGE_PASSWORD: ${MESSAGE_PASSWORD}
+      MESSAGES_TOPIC_ADDRESS: fcp-fd-comms-${DEV_SUFFIX}
+      MESSAGES_SUBSCRIPTION_ADDRESS: fcp-fd-comms-${DEV_SUFFIX}
+      COMMS_AZURE_CLIENT_ID: ${COMMS_AZURE_CLIENT_ID}
+      NOTIFY_API_KEY: ${NOTIFY_API_KEY}
+      NOTIFY_TEST_EMAIL: ${NOTIFY_TEST_EMAIL}
+      CONFIRM_NEW_USER_NOTIFY_TEMPLATE_ID: ${CONFIRM_NEW_USER_NOTIFY_TEMPLATE_ID}
```

Comment:
Don't think we need reference to the template in env vars?

Reply:
Have now removed this.